components/sbm-recipes-boot-upgrade/src/test/java/org/springframework/sbm/boot/upgrade_27_30/report/SpringBootUpgradeReportActionTest.java

```diff
@@ -160,7 +160,7 @@ void verifyRenderedHtml(@TempDir Path tempDir) throws IOException {
                     <parent>
                         <groupId>org.springframework.boot</groupId>
                         <artifactId>spring-boot-starter-parent</artifactId>
-                        <version>2.7.3</version>
+                        <version>2.7.2</version>
```

Comment:
What made you change this?

Reply:
I'm using a machine that hasn't direct access to Maven Central, and on the Artifactory instance I'm using, this artifact is flagged, so I downgraded it.

Reply:
OK ;) 
It doesn't hurt, all tests still passing.